File: app/src/main/java/net/imatruck/dsmanager/models/DSGetInfoData.java
package net.imatruck.dsmanager.models;

import com.google.gson.annotations.SerializedName;


@SuppressWarnings("unused")
public class DSGetInfoData {
    @SerializedName("is_manager")
    private boolean isManager;
    private int version;
    @SerializedName("version_string")
    private String versionString;

    public boolean isManager() {
        return isManager;
    }

    public int getVersion() {
        return version;
    }

    public String getVersionString() {
        return versionString;
    }
}

File: app/src/main/java/net/imatruck/dsmanager/models/DSGetInfoData.kt
package net.imatruck.dsmanager.models

import com.google.gson.annotations.SerializedName


class DSGetInfoData {
    @SerializedName("is_manager")
    val isManager: Boolean = false
    val version: Int = 0
    @SerializedName("version_string")
    val versionString: String? = null
}
